sdk/dapp-kit/src/hooks/wallet/useUnsafeBurnerWallet.ts

```diff
@@ -128,15 +132,36 @@ function registerUnsafeBurnerWallet(suiClient: SuiClient) {
 			};
 		};
 
-		#signAndExecuteTransactionBlock: SuiSignAndExecuteTransactionBlockMethod = async (
+		#signAndExecuteTransactionBlock: SuiSignAndExecuteTransactionBlockV2Method = async (
 			transactionInput,
 		) => {
-			return await suiClient.signAndExecuteTransactionBlock({
-				signer: keypair,
-				transactionBlock: transactionInput.transactionBlock,
-				options: transactionInput.options,
-				requestType: transactionInput.requestType,
-			});
+			const { rawEffects, balanceChanges, digest } = await suiClient.signAndExecuteTransactionBlock(
+				{
+					signer: keypair,
+					transactionBlock: TransactionBlock.from(transactionInput.transactionBlock),
+					options: {
+						showRawEffects: true,
+						showBalanceChanges: true,
+					},
+				},
+			);
+
+			return {
+				digest,
+				effects: rawEffects ? toB64(new Uint8Array(rawEffects)) : null,
```

Comment:
Curious what it's returned as from the API?

Reply:
`number[]`